src/CppUTestExt/MockFailure.cpp

```diff
@@ -104,10 +104,15 @@ MockExpectedCallsDidntHappenFailure::MockExpectedCallsDidntHappenFailure(UtestSh
 MockUnexpectedCallHappenedFailure::MockUnexpectedCallHappenedFailure(UtestShell* test, const SimpleString& name, const MockExpectedCallsList& expectations) : MockFailure(test)
 {
     int amountOfExpectations = expectations.amountOfExpectationsFor(name);
-    if (amountOfExpectations)
```

Comment:
I'd agree, I'd prefer not adding useless local variables, that doesn't really improve readibility.